### ISSUE TITLE
Bug: single duplicate in some collections

### ISSUE THREAD

**Issue by mergen3107** [stelzch/zotero.koplugin#16]
Hi @stelzch 
I found out that **some** of the collections after syncing has only one duplicate. I don't know what's the pattern, it seems a little random.

Trying to come up with a good example.
Here is the items.json and collestions.json: 
[zotero_duplicates.zip](https://github.com/user-attachments/files/15933613/zotero_duplicates.zip)

Duplicate in `MAIN COLLECTION`:
<img src= "https://github.com/stelzch/zotero.koplugin/assets/6279855/8b88b607-eee2-49d4-a665-acc852680504" height="600">

Duplicate in `SOLAR CYCLE IBEX`:
<img src= "https://github.com/stelzch/zotero.koplugin/assets/6279855/02b8f0fe-f8d9-4e14-ae41-52d569cf65b9" height="600">

However, no duplicate in `Voyager 1 HP Crossing`:
<img src= "https://github.com/stelzch/zotero.koplugin/assets/6279855/c1d416e4-bcbc-4505-8b88-9ec2aac7d7bc" height="600">

**Comment by mergen3107** [comment]
Both of the duplicates work OK! If I click them, they open up no problem. So this is just a cosmetic bug.

**Comment by mergen3107** [comment]
1) I found out that one of my articles had two attachments under the same metadata name. So I deleted it and synced again. However, the duplicate in that collection stayed!

2) I deleted koreader/zotero folder and started fresh with used_id and API_key.
I don't see any duplicates anymore!
That means this problem is the same as https://github.com/stelzch/zotero.koplugin/issues/15 .

Closing this issue in favor of that.
